notifications_utils/template.py

```diff
@@ -158,8 +158,28 @@ def __init__(
         self.prefix = prefix
         self.show_prefix = show_prefix
         self.sender = sender
+        self._content_count = None
         super().__init__(template, values)
 
+    @property
+    def values(self):
+        return super().values
+
+    @values.setter
+    def values(self, value):
+
+        # If we change the values of the template it’s possible the
+        # content count will have changed, so we need to reset the
+        # cached count.
+        if self._content_count is not None:
+            self._content_count = None
+
+        # Assigning to super().values doesn’t work here. We need to get
```

Comment:
Optional: Nice code comment but I'd also like to know why assigning to super().values doesn't work, rather than just the fact that it doesn't please?

Reply:
The specific error is
```
AttributeError: 'super' object has no attribute 'value'
```

There’s a bug report here: https://bugs.python.org/issue14965

Or as [someone on Stackoverflow put it](https://stackoverflow.com/a/10810545/147318):

> I was trying to find the correct language to back up why this behavior is the way it is, so as not to give you a "because it just is" answer...